tests/framework/common.go

```diff
@@ -1196,9 +1196,14 @@ func (td *OsmTestData) Cleanup(ct CleanupType) {
 	if td.CollectLogs == CollectLogs || td.CollectLogs == ControlPlaneOnly ||
 		((restartSeen && !td.IgnoreRestarts) || CurrentGinkgoTestDescription().Failed) && td.CollectLogs == CollectLogsIfErrorOnly {
 		// Grab logs. We will move this to use CLI when able.
+
 		if err := td.GrabLogs(); err != nil {
```

Comment:
`bug-report` should be capturing all data that `GrabLogs` collects (including proxy debug paths and namespace events). If that is true then we should be safe to remove `GrabLogs`, as it'll just be redundant.

Reply:
Ok makes sense. We should add that to `bug-report` so it can be a full solution for diagnostic logs.